modules/persistent-storage-csi-smb-cifs-provision-static.adoc

```diff
@@ -74,15 +74,15 @@ spec:
     - file_mode=0777
   csi:
     driver: smb.csi.k8s.io
-    volumeHandle: smb-server.default.svc.cluster.local/share/ <2>
+    volumeHandle: smb-server.default.svc.cluster.local/share#subdir#app-data <2>
```

Comment:
What is "app-data" and where it came from?

Reply:
Just follow the `volumeHandle` fomat, `app-data`(share-name) seems useless in static provision.